src/ripe/order.py

```diff
@@ -27,6 +27,246 @@ def get_order(self, number):
+        url = self.base_url + "orders/%d/attachments/%d" % (number, attachment_id)
+        contents = self.get(url)
+        return contents
+
+    def create_note_order(self, number, note):
+        url = self.base_url + "orders/%d/notes" % number
+        contents = self.post(url, data_j = note)
+        return contents
+
+    def create_waybill_order(self, number, waybill):
+        url = self.base_url + "orders/%d/waybill" % number
+        contents = self.post(url, data_j = waybill)
+        return contents
+
+    def refresh_shipping_order(self, number):
+        url = self.base_url + "orders/%d/refresh_shipping" % number
+        contents = self.post(url)
+        return contents
+
+    def log_order(self, number):
+        url = self.base_url + "orders/%d/log" % number
+        contents = self.get(url)
+        return contents
+
+    def states_order(self, number):
+        url = self.base_url + "orders/%d/states" % number
+        contents = self.get(url)
+        return contents
+
+    def state_order(self, number, state_id):
+        url = self.base_url + "orders/%d/states/%d" % (number, state_id)
+        contents = self.get(url)
+        return contents
+
+    def state_chat_order(self, number, state_id):
+        url = self.base_url + "orders/%d/states/%d/chat" % (number, state_id)
+        contents = self.get(url)
+        return contents
+
+    def state_chat_lines_order(self, number, state_id):
+        url = self.base_url + "orders/%d/states/%d/chat/lines" % (number, state_id)
+        contents = self.get(url)
+        return contents
+
+    def state_chat_lines_count_order(self, number, state_id):
+        url = self.base_url + "orders/%d/states/%d/chat/lines/count" % (number, state_id)
+        contents = self.get(url)
+        return contents
+
+    def state_chat_create_line_order(self, number, state_id, content):
+        url = self.base_url + "orders/%d/states/%d/chat/lines/count" % (number, state_id)
+        contents = self.post(url, data_j = content)
+        return contents
+
+    def state_attachments_order(self, number, state_id):
+        url = self.base_url + "orders/%d/states/%d/attachments" % (number, state_id)
+        contents = self.get(url)
+        return contents
+
+    def state_attachments_count_order(self, number, state_id):
+        url = self.base_url + "orders/%d/states/%d/attachments/count" % (number, state_id)
+        contents = self.get(url)
+        return contents
+
+    def state_create_attachments_order(self, number, state_id, file):
+        url = self.base_url + "orders/%d/states/%d/attachments" % (number, state_id)
+        contents = self.post(url, data_j = file)
+        return contents
+
+    def state_attachment_order(self, number, state_id, attachment_id):
+        url = self.base_url + "orders/%d/states/%d/attachments/%d" % (number, state_id, attachment_id)
+        contents = self.get(url)
+        return contents
+
+    def quality_assure_order(

```

Comment:
Please be aware of proper sorting of methods, the quality assure should be together with the ready, produce, etc. methods